.clang-format

```diff
@@ -0,0 +1,92 @@
+# This configuration file can be used to auto-format the code base
+# To apply the coding style you can run the following command (assuming you installed clang-format-6.0 on your system):
+# $ find ./src -iregex '.*\.\(cpp\|hpp\|c\|h\|cc\|hh\|cxx\|hxx\)$' | xargs clang-format-6.0 -i -style=file
+# You can find more information on the different config parameters in this file here:
+# https://releases.llvm.org/6.0.0/tools/clang/docs/ClangFormatStyleOptions.html
+---
+TabWidth: 2
```

Comment:
this looks wrong

Reply:
The TabWidth parameter says how much colums (spaces) are equivalent to one tab. since no tabs are used this parameter has no effect. 
I put all clang-format known parameters in this file. So it is easier to tweak the parameters.